debian/patches/test--skip_TestFactoryNewTmpfs.patch

```diff
@@ -5,13 +5,11 @@ Description: disable test (requires root)
 
 --- a/libcontainer/factory_linux_test.go
 +++ b/libcontainer/factory_linux_test.go
-@@ -76,8 +76,9 @@
- 	}
+@@ -78,6 +78,7 @@
  }
  
  func TestFactoryNewTmpfs(t *testing.T) {
 +t.Skip("DM - skipping privileged test")
```

Comment:
These test files end up getting shipped in the `-dev` package, right?  I think skipping the tests is probably fine, but it might make sense to make sure the result can pass through `gofmt` cleanly (as far as I can see, indentation is the only issue here and in one of the other patches).

Reply:
As I understood those tests pass but not in the build environment, and about the indentation I did not change anything, just kept the patches as they were before. Would you like me to fix the indentation?

Reply:
Ahh, hmm.  In that case, I think it's fine -- it would probably be good to somehow skip these only for `debian/rules` testing instead (so we could have autopkgtests run them), but I don't think it's important enough that we should spend a lot of effort on it (especially since `autopkgtests` does verification of the end-to-end process within Docker and containerd anyway).

Reply:
Then I am marking this as solved.